### PR TITLE
Adding very large tensor tests for JIT kernels

### PR DESCRIPTION
**Pull request by kushanam**
This is to test adding large tensor tests for jit kernel path.
CC: @frgossen

### CODE REVIEW

File: tensorflow/core/kernels/mlir_generated/base_ops_test.h [hunk 1]
```diff
@@ -199,6 +199,16 @@ absl::InlinedVector<T, 10> DefaultInputNonZero() {
                                          0.2, 0.3, 0.5, 0.7, 0.9, 9.0, 18.0});
 }
 
+template <typename T, std::enable_if_t<
+                          llvm::is_one_of<T, Eigen::half, float, double>::value,
+                          bool> = true>
+absl::InlinedVector<T, 10> VeryLargeVector() {
+  auto max_val = pow(2,32) + 1;
```
Comment (akuegel) [inline tensorflow/core/kernels/mlir_generated/base_ops_test.h:206]: I am not sure whether we want to have a test that has 2^32 + 1 many elements. Do we really need an end-to-end test for this? I also see that you have currently commented out the test that uses this function.

@frgossen what do you think?

**Review by akuegel — COMMENTED** [review]
The tf2tensorrt changes seem unrelated to adding the very large tensor tests.
If this is true, I would suggest to split this into two pull requests.
Also, in my opinion it would be better not to add an end2end test for very large tensors, and instead test the codegen for this case (but possibly there already exists already tests for that).

**Comment by gbaned** [comment]
@kushanam Can you please check @akuegel's @frgossen's comments and resolve conflicts?. Thank you!

**Review by frgossen — CHANGES_REQUESTED** [review]
Is this PR still active or shall we close it in favour of https://github.com/tensorflow/tensorflow/pull/58045?

File: tensorflow/core/kernels/mlir_generated/base_ops_test.h (hunk 1 above)
Comment (frgossen) [reply tensorflow/core/kernels/mlir_generated/base_ops_test.h:206]: Yes, we have codegen tests. I think, the e2e test should exist somewhere. We can always disable it if it is too expensive.

**Pull request closed without merging**